src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -0,0 +1,67 @@
+package mate.jdbc.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id).orElse(new Car());
+    }
+
+    @Override
+    public List<Car> getAll() {
+        return carDao.getAll();
+    }
+
+    @Override
+    public Car update(Car car) {
+        return carDao.update(car);
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        return carDao.delete(id);
+    }
+
+    @Override
+    public void addDriverToCar(Driver driver, Car car) {
+        car.getDrivers().add(driver);
+        carDao.update(car);
+    }
+
+    @Override
+    public void removeDriverFromCar(Driver driver, Car car) {
+        car.getDrivers().remove(driver);
+        carDao.update(car);
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        List<Car> result = new ArrayList<>();
+        List<Car> carList = carDao.getAll();
+        for (Car car : carList) {
+            for (Driver driver : car.getDrivers()) {
+                if (Objects.equals(driver.getId(), driverId)) {
+                    result.add(car);
+                }
+            }
+        }
+        return result;
+    }
```

Comment:
This method should be implemented in CarDao. It is written in readme.md that only 2 methods( `public void removeDriverFromCar(Driver driver, Car car)` AND `public void addDriverToCar(Driver driver, Car car)`) should NOT be implemented in CarDao. Other methods should.